sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2309,8 +2344,11 @@ object Radar {
         logger.i("📍️ Radar error received | status = $status")
     }
 
-    internal fun sendLog(message: String) {
+    internal fun sendLog(level: RadarLogLevel, message: String) {
         receiver?.onLog(context, message)
+        if (isTestKey()) {
+            logBuffer.write(level, message)
```

Comment:
do we need to check if this is initialized before calling? (also curious why `this.logBuffer` isn't required here, because it's an `internal fun`?)

Reply:
It's not needed. The only place that calls `sendLog` is in `RadarLogger`, which is initialized in the `Radar.initialize` method, only after the log buffer is initialized.